22bca38.ipynb

```diff
@@ -0,0 +1,92 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "27232e2d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "10ba538b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def getstaf_record():\n",
+    "    sid=int(input(\"enter staf id\"))\n",
+    "    sname=input(\"staf name\")\n",
+    "    salary=int(input(\"enter salary\"))\n",
+    "    return[sid,sname,salary]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "6a93a68c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "record=[\n",
+    "    [1,'Kevin',5000],\n",
+    "    [2,'om',4000],\n",
+    "    [3,'Rudra',6000],\n",
+    "    [4,'Raj',3000],\n",
+    "    [5,'sai',4200]\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "e09df8a5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open(\"staf_record.csv\",\"w\",newline='') as file:\n",
+    "    writer=csv.writer(file)\n",
+    "    writer.writerow([\"sid\",\"sname\",\"salary\"])\n",
```

Comment:
please can you explain me ,what is the difference between writerow and writerows......?

Reply:
Writerow is use for single line and writerows is use for multiple line.

Reply:
Writerow is use for single line and writerows is use for multiple line.